lib/code_corps_web/controllers/comment_controller.ex

```diff
@@ -1,30 +1,41 @@
 defmodule CodeCorpsWeb.CommentController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2]
+  alias CodeCorps.{Comment, User, Helpers.Query}
 
-  alias CodeCorps.Comment
+  action_fallback CodeCorpsWeb.FallbackController
```

Comment:
This could go into the `code_corps_web.ex` `using` block once we make the full switch.

Any additional fallbacks can be put into the `FallbackController` module if they are used across controllers, or into the current controller module as a function, if specific to this controller, since this is just a plug anyway.